examples/alm.jl

```diff
@@ -0,0 +1,48 @@
+using FrankWolfe
+using ProgressMeter
+using Arpack
+using DoubleFloats
+using ReverseDiff
+using LinearAlgebra
+
+n = Int(1e4)
+
+xpi = rand(1:100, n)
+total = sum(xpi)
+xp = xpi .// total
+
+f(x) = norm(x - xp)^2
+
+function grad!(storage, x)
+    @. storage = 2 * (x - xp)
+end
+
+vertices = 2*rand(100, n) .- 1
+
+lmoNB = FrankWolfe.ScaledBoundL1NormBall(-ones(n), ones(n))
```

Comment:
```suggestion
lmo_nb = FrankWolfe.ScaledBoundL1NormBall(-ones(n), ones(n))
```

for variables and fields, snake case is preferred. CamelCase like this is kept for types. Same for lmoProb below